package.json

```diff
@@ -11,10 +11,15 @@
 		"build": "tsc -p src",
 		"dev": "yarn build && node .",
 		"clean": "rimraf dist",
-		"watch": "tsc -b src -w",
+		"watch": "tsc-watch -b src --onSuccess \"node .\"",
```

Comment:
added `tsc-watch` so you can watch & run at the same time without needing 2 processes.